torch/csrc/autograd/functions/tensor.cpp

```diff
@@ -77,7 +77,7 @@ auto CopySlices::apply(variable_list&& inputs) -> variable_list {
       AT_ASSERT(res[i].defined());
       if (i == 0) {
         grad_slice.copy_(res[i]);
-        grad_inputs[i] = std::move(result);
+        grad_inputs[i] = std::move(result); // NOLINT(bugprone-use-after-move)
```

Comment:
Huh, I wonder why the lint gets this wrong.

Reply:
static analysis is hard ...